modules/lb-http-backend/metadata.yaml

```diff
@@ -0,0 +1,315 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: blueprints.cloud.google.com/v1alpha1
+kind: BlueprintMetadata
+metadata:
+  name: terraform-google-lb-http-lb-http-backend
+  annotations:
+    config.kubernetes.io/local-config: "true"
+spec:
+  info:
+    title: HTTP Load balancer backend module
+    source:
+      repo: https://github.com/terraform-google-modules/terraform-google-lb-http.git
+      sourceType: git
+      dir: /modules/lb-http-backend
```

Comment:
WDYT about calling this submodule just `backend` (and the other frontend)?

I feel like `lb-http` prefix is not necessary since these are submodules of terraform-google-lb-http.

Reply:
Sounds good! Done.